blackbox/docs/sql/administration/privileges.txt

```diff
@@ -73,20 +76,20 @@ If a user with the username specified in the SQL statement does not exist the
 statement returns an error::
 
     cr> GRANT DQL TO layla;
-    UserDoesNotExistException ...
+    SQLActionException[SQLParseException: User layla does not exists]
```

Comment:
hm this shouldn't be an parse exception but more an unknown one, can you pls change that?